### DISCUSSION TITLE
Too many queries from spoolman in pihole

### DISCUSSION THREAD

**Discussion by itsmemac** [Donkie/Spoolman#309] [Q&A]
Hi everyone

I have an instance of spoolman running in my Unraid server for couple of weeks and have 2 pihole in the network (1 in Unraid and another in Rpi)
I started seeing more number of queries from spoolman instance for the last couple of weeks

is this ordinary ?

Have attached a screenshot which shows over 84k queries being processed by pihole from spoolman
This pihole is the 2nd instance hence only works when 1st one is down

Spoolman version : Version 0.16.1 (babb95d)
System : Docker (Unraid container)

![image](https://github.com/Donkie/Spoolman/assets/37410297/21f0bdc6-3ca7-4e21-8cdf-7aeade8c2daa)

**Comment by vikt-v** [comment]
Aren't those queries looking to resolve spoolman address?
I suspect those queries are originating from Fluid, Mainstail or some other integration.

**Reply by itsmemac** [reply]
my printer has been down for a month and this issue started happening only after recent update of spoolman

```
Feb 10 12:23:58: query[A] spool.example.com from 127.0.0.1
Feb 10 12:23:58: cached spool.example.com is <CNAME>
Feb 10 12:23:58: cached example.com is 10.20.30.2
Feb 10 12:23:58: query[AAAA] spool.example.com from 127.0.0.1
Feb 10 12:23:58: cached spool.example.com is <CNAME>
Feb 10 12:23:58: cached example.com is NODATA-IPv6
Feb 10 12:24:00: query[A] spool.example.com from 127.0.0.1
Feb 10 12:24:00: cached spool.example.com is <CNAME>
Feb 10 12:24:00: cached example.com is 10.20.30.2
```

**Comment by Donkie** [comment]
This is not really a Spoolman issue. This happens because you've configured your Moonraker/OctoPrint to point to spoolman using the domain name you've created (spool.example.com). Either make sure the DNS is cached locally on the moonraker/octoprint server, or point it to the IP:Port of the Spoolman server instead.
